<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBanks</class>
 <widget class="QWidget" name="QBanks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pshBut_bybank">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>460</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bank</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_operation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Operation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sort_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Style :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshBut_bystyle">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Style</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_sort_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>464</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshBut_exchange">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exchange</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sort">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sort by :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshBut_byname">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_bank">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPlainTextEdit" name="txtEdit_voicename">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="maximumBlockCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pshBut_copy">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>490</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Duplicate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbBox_voicestyle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Piano</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Keys</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Organ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Guitar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bass</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orch</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Brass</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Synth</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ethnic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bells</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rythm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SFX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
